### PR TITLE
Hide links in dropdown when page is smol

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/jvmMain/sass/main.scss [hunk 1]
```diff
@@ -1591,6 +1647,36 @@ small.alert-count {
       width: 44px;
     }
   }
+
+  .thin-dd {
+    .dropdown-divider {
+      display: none;
```
Comment (Pixelguymm) [inline src/jvmMain/sass/main.scss:1653]: I'd consider making these visible on larger screens as well, as vertical dividers that is. If you don't agree or want to leave that for another PR, everything else looks good to me.

**Review by Pixelguymm — APPROVED** [review]

File: src/jvmMain/sass/main.scss (hunk 1 above)
Comment (Top-Cat) [reply src/jvmMain/sass/main.scss:1653]: Can leave that for the future
